### ISSUE TITLE
Failed to build kompose source code

### ISSUE THREAD

**Issue by janetkuo** [skippbox/kompose#28]
Build failed on HEAD. Not sure if I'm doing anything wrong. 

Building with Docker doesn't work:
```
$ make binary 
docker build -t "libcompose-dev:kcompose" .
Sending build context to Docker daemon 16.91 MB
Step 1 : FROM golang:1.5.2
 ---> 29d67fe18deb
Step 2 : RUN apt-get update && apt-get install -y     iptables     build-essential     --no-install-recommends
 ---> Using cache
 ---> ba4ff11ec431
Step 3 : RUN go get github.com/mitchellh/gox
 ---> Using cache
 ---> ae61dd93f744
Step 4 : RUN go get github.com/aktau/github-release
 ---> Using cache
 ---> ec126f459a35
Step 5 : RUN go get golang.org/x/tools/cmd/cover
 ---> Using cache
 ---> a07d2c254ff2
Step 6 : RUN go get github.com/golang/lint/golint
 ---> Using cache
 ---> eb11c5ec5796
Step 7 : RUN go get golang.org/x/tools/cmd/vet
 ---> Running in 98eb2c545a6f
package golang.org/x/tools/cmd/vet: cannot find package "golang.org/x/tools/cmd/vet" in any of:
	/usr/local/go/src/golang.org/x/tools/cmd/vet (from $GOROOT)
	/go/src/golang.org/x/tools/cmd/vet (from $GOPATH)
The command '/bin/sh -c go get golang.org/x/tools/cmd/vet' returned a non-zero code: 1
make: *** [build] Error 1
```

Building with go doesn't work either:
```
$ echo $GO15VENDOREXPERIMENT
1
$ go build -o libcompose ./cli/main
# github.com/skippbox/kompose/cli/main
cli/main/main.go:22: cannot use "github.com/docker/libcompose/cli/app".BeforeApp (type func(*"github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Context) error) as type func(*"github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Context) error in assignment
cli/main/main.go:23: cannot use append(command.CommonFlags(), "github.com/docker/libcompose/cli/docker/app".DockerClientFlags()...) (type []"github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Flag) as type []"github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Flag in assignment
cli/main/main.go:25: cannot use command.BuildCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:26: cannot use command.CreateCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:27: cannot use command.UpCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:28: cannot use command.StartCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:29: cannot use command.LogsCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:30: cannot use command.RestartCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:31: cannot use command.StopCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:32: cannot use command.ScaleCommand(factory) (type "github.com/docker/libcompose/vendor/github.com/codegangsta/cli".Command) as type "github.com/skippbox/kompose/vendor/github.com/codegangsta/cli".Command in array element
cli/main/main.go:32: too many errors
```

Docker version: 1.9.1, go version: 1.5.1, OS: linux amd64

**Comment by ngtuna** [comment]
@janetkuo: I encourage you to build with Docker by running `make binary` at this moment. However, just have a quick check, package "golang.org/x/tools/cmd/vet" has been removed from [master branch](https://github.com/golang/tools/tree/master/cmd). That's why we can't get it down...

**Comment by ngtuna** [comment]
I will move these dependencies into `vendoring`

**Comment by ngtuna** [comment]
@janetkuo now you can build kcompose with Docker. I updated dependencies and moved package "golang.org/x/tools/cmd/vet" to `vendor` folder. Fixed 84fa2e06397eaa1ff5a8425053969dbd0c52d3c9
